syntax = "proto3";

service VideoStreamService {
  rpc PlayVideo (VideoRequest) returns (stream VideoChunk);
}

message VideoRequest {
  string video_path = 1;
}

message VideoChunk {
  bytes data = 1;
  int32 frame_index = 2;
}
